react/features/conference/components/native/Conference.js

```diff
@@ -42,6 +42,13 @@ import type { AbstractProps } from '../AbstractConference';
  */
 type Props = AbstractProps & {
 
+    /**
+     * Wherther the calendar feasture is enabled or not.
```

Comment:
a typo in `feasture`
looks like your hand slips the same way whenever types this word :)

Reply:
FML. I blame the new apple keyboard! :-P